examples/ts-node/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "subspace-js-node-example",
+  "version": "0.1.0",
+  "scripts": {
+    "build": "rm -rf dist && tsc",
+    "start": "node ./dist/index.js"
+  },
+  "dependencies": {
+    "subspace.js": "file:../../../subspace.js"
+  },
+  "devDependencies": {
+    "dotenv": "^10.0.0",
```

Comment:
This is used in runtime, so it shouldn't be in dev dependencies. Also I think it would be better to hardcode URLs in code such that people can see what those URLs are and can modify it right there. The simpler it is - the better.

Reply:
I can hardcode these constants and remove the `.env` file and `dotenv` dependency.

```javascript
const NODE_WS_PROVIDER = "ws://localhost:9944";
const FARMER_WS_PROVIDER = "ws://localhost:9955";
const suri = "//Alice";
```